compiler/luci-interpreter/pal/mcu/pal.cmake

```diff
@@ -0,0 +1,49 @@
+macro(initialize_pal)
+    nnas_find_package(TensorFlowSource EXACT 2.3.0 QUIET)
+    nnas_find_package(TensorFlowGEMMLowpSource EXACT 2.3.0 QUIET)
```

Comment:
I am not sure MCU mode needs this library, maybe add something like `#todo check if this is needed`?